.github/SECURITY.md

```diff
@@ -0,0 +1,22 @@
+# Security Policy
+
+## Supported Versions
+
+| Version | Supported          |
+| ------- | ------------------ |
+| 5.0.x   | :white_check_mark: |
```

Comment:
Refer to the releases page. Only the most recent stable version is supported (in general).